cmd/gnodev/test.go

```diff
@@ -298,17 +292,8 @@ func gnoTestPkg(
 			if err != nil {
 				errs = multierr.Append(errs, err)
 				io.ErrPrintfln("--- FAIL: %s (%s)", testName, dstr)
-				if verbose {
-					stdouterr, err := closer()
```

Comment:
this was causing problem (by definition when it reached here because of line 288-291 `closer` was nil, causing a segfault because calling a 0x0000 pointer, the reason it was not apparent is because it only was called for _testfile AND when verbose AND when there was some error).